eng/common/scripts/Update-DocsMsMetadata.ps1

```diff
@@ -114,16 +114,15 @@ function GetAdjustedReadmeContent($ReadmeContent, $PackageInfo, $PackageMetadata
   if ($codeOwnerArray) {
     Write-Host "Code Owners are $($codeOwnerArray -join ",")"
     $author = $codeOwnerArray[0]
-    try {
-      $msauthor = ."$PSScriptRoot/Get-AADIdentityFromGithubUser.ps1" `
-                    -TenantId $TenantId `
-                    -ClientId $ClientId `
-                    -ClientSecret $ClientSecret `
-                    -GithubUser $author
-    }
-    catch {
-      LogError $_
+    
+    $msauthor = ."$PSScriptRoot/Get-AADIdentityFromGithubUser.ps1" `
```

Comment:
If this ps script is the only place we call this we might want to consider defining a helper function to map the github user to make it easier for consuming (i.e. no need to check exit codes and only check the return result).